atest/robot/parsing/suite_names.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+Documentation     Run testdata and validate that suite names are set correctly
+Suite Setup     Run Tests    ${EMPTY}    misc/suites
+Resource        atest_resource.robot
+# Test Tags       suite_naming
```

Comment:
1. Please remove the commented Test Tags line. No need to have such tags and we avoid commented out code as well.
2. Align setting values.

Reply:
tag removed
settings aligned